.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+snapshot.jpeg
```

Comment:
This is going to have to change to `env.py` but it works for now until we actually implement the module.